docs/reference/rest-api/cron-expressions.asciidoc

```diff
@@ -0,0 +1,182 @@
+[[cron-expressions]]
+=== Cron expressions
+
+{es} uses the cron parser from the http://www.quartz-scheduler.org[Quartz Job Scheduler]. For more
+information about writing Quartz cron expressions, see the
+http://www.quartz-scheduler.org/documentation/quartz-2.2.x/tutorials/tutorial-lesson-06.html[Quartz CronTrigger Tutorial].
+
+All schedule times are in UTC, other timezones are not supported.
+
+A cron expression is a string of the following form:
+
+[source,txt]
+------------------------------
+    <seconds> <minutes> <hours> <day_of_month> <month> <day_of_week> [year]
+------------------------------
+
+TIP: You can use the <<elasticsearch-croneval>> command line tool to validate your cron expressions.
+
+
+[[cron-elements]]
+==== Cron expression elements
+
+All elements are required except for `year`. 
+See <<cron-special-characters>> for information about the allowed special characters.
+
+seconds::
+(Required)
+Valid values: 0-59 and the special characters `,` `-` `*` `/`
+
+minutes:: 
+(Required)
+Valid values: 0-59 and the special characters `,` `-` `*` `/`
+
+hours:: 
+(Required)
+Valid values: 0-23 and the special characters `,` `-` `*` `/`
+
+day_of_month:: 
+(Required)
+Valid values: 1-31 and the special characters `,` `-` `*` `/` `?` `L` `W`
+
+month::  
+(Required)
+Valid values: 1-12, JAN-DEC, jan-dec, and the special characters `,` `-` `*` `/`
+
+day_of_week:: 
+(Required)
+Valid values: 1-7, SUN-SAT, sun-sat,  and the special characters `,` `-` `*` `/` `?` `L` `#`
+
+year:: 
+(Optional)
+Valid values: 1970-2099 and the special characters `,` `-` `*` `/`
+
+[[cron-special-characters]]
+==== Cron special characters
+
+*::
+All values. Selects every possible value for a field. For
```

Comment:
Not sure if both statements are needed. I think it's less confusing without the first.
```suggestion
Selects every possible value for a field. For
```